package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@i-cell/data-table",
-  "version": "2.0.33",
+  "version": "2.0.35",
```

Comment:
`2.0.34` set it to the current not the one you want :)

Reply:
okay :)

Reply:
done